src/Feature/Hover/Feature_Hover.re

```diff
@@ -0,0 +1,28 @@
+/* Feature_Hover.re
+     This feature project contains logic related to Hover
+   */
+
+[@deriving show({with_path: false})]
+type command =
+  | Show
+  | Hide;
+
+module Commands = {
+  open Feature_Commands.Schema;
+
+  let show =
+    define(
+      ~category="Hover",
+      ~title="Show hover panel",
+      "hover.show",
```

Comment:
For the command IDs, we should try and keep these in sync with VSCode - it's important for an extension compatibility, because some extensions might try and show hover. 

It looks like the equivalent is `"editor.action.showHover"`: https://github.com/microsoft/vscode/blob/eff2fb75151dac67b6835584b6e91808ec1379ae/src/vs/editor/contrib/hover/hover.ts#L234

Reply:
Oh okay, I didn't even realize those were based in VSCode command ID's 😳. Thanks for the tip!

Reply:
Fixed in the latest commit! It also looks like they dont have a hide. From what I can tell they hide when a non-modifier key is pressed. I got rid of the `Hide` command too.